### DISCUSSION TITLE
Lower Pressure than expected through damper

### DISCUSSION THREAD

**Discussion by FPSSPF** [firemodels/fds#13550] [General]
Hello All, 
I am modelling two enclosures connected by means of a damper. The damper has an area of 0.2m2. The loss in the damper is fixed as 0.55 in both directions. Roughtness is set as 0. Both enclosures are connected to the outside by means of ducts modelled with HVAC. In one of the ducts I have prescribed an extrac rate of 2m3/s. This extract rate is reached after at 10s.

Then, it is expected that the air from the outside enterns enclosure 1, goes trough the damper with a velocity of 4m/s (2m3/s / 0.2m2) and extracted in enclosure 2. I would expect a pressure drop through the damper of 5.28Pa (iP=(/rho*k*v*v)/2). However, at 20s I am observing a pressure diference between the rooms of 60Pa approx.

I think the difference of pressure between rooms should be approx. 5Pa. Is there any problem in my model or I am not seeing something super obvious?

I suspect that the problem relates with the pressure zones.

I attach the .fds file.
[TestP.txt](https://github.com/user-attachments/files/17309746/TestP.txt)
 

![image](https://github.com/user-attachments/assets/47495e47-75eb-4f14-9b72-c81657ba5a6b)

**Comment by mcgratta** [comment]
Nothing is super obvious when your input file is hundreds of lines long. In a case like this, it is best to simplify the case as much as possible. Leave only the essential components and output.

**Reply by drjfloyd** [reply]
2 m3/s/0.2 m^2 is 10 m/s, not 4 m/s.  The expected pressure drop is going to 0.5 * 0.55 / 1.2 * 10^2 = 23 Pa.

Make sure you are looking at the pressures as seen by the HVAC nodes. Add NODE PRESSURE devices to get this.

**Reply by drjfloyd** [reply]
edit that should be K rho not K / rho so ~33 Pa.

Ran your case until steady state with NODE PRESSURE and VOLUME FLOW for that duct. Getting dP ~ 33 Pa and 2 m3/s flow.  

![image](https://github.com/user-attachments/assets/a07d6540-eae7-424b-8a18-85318845c90b)

**Reply by FPSSPF** [reply]
Hello drfloyd, 

You are completely right. When preparing my query I mixed several things. My fault. 

In any case I understand from where comes my ´issue´ and it is not the damper modelling.

Thanks!
